spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/SystemFunctionPushDownHelper.java

```diff
@@ -39,6 +63,22 @@ public static void createPartitionedTable(
     insertRecords(spark, tableName);
   }
 
+  public static void createPartitionedTable(
```

Comment:
I don't think these additional versions are needed since the tests in this PR should not need to exercise different underlying data formats.